In-house patch to remove unnecessary dependencies from Ironic's
requirements files. The specific reasons are as follows:

lxml			Not applicable

oslo.rootwrap		Not applicable to Solaris

posix-ipc		Not applicable

sqlalchemy-migrate	Not applicable

websockify		Not applicable

--- ironic-2015.1.2/ironic.egg-info/requires.txt.~1~	2015-10-13 10:59:16.000000000 -0700
+++ ironic-2015.1.2/ironic.egg-info/requires.txt	2016-02-01 20:03:06.190153642 -0800
@@ -2,10 +2,8 @@ pbr!=0.7,<1.0,>=0.6
 SQLAlchemy<=0.9.99,>=0.9.7
 alembic<0.8.1,>=0.7.2
 eventlet!=0.17.0,>=0.16.1
-lxml>=2.3
 WebOb>=1.2.3
 greenlet>=0.3.2
-sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 netaddr>=0.7.12
 paramiko>=1.13.0
 iso8601>=0.1.9
@@ -15,12 +13,10 @@ python-keystoneclient<1.4.0,>=1.2.0
 python-swiftclient<2.5.0,>=2.2.0
 stevedore<1.4.0,>=1.3.0 # Apache-2.0
 pysendfile==2.0.0
-websockify<0.7,>=0.6.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.context<0.3.0,>=0.2.0 # Apache-2.0
 oslo.db<1.8.0,>=1.7.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 oslo.policy<0.4.0,>=0.3.1 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
@@ -34,4 +30,3 @@ Jinja2>=2.6 # BSD License3 clause
 keystonemiddleware<1.6.0,>=1.5.0
 oslo.messaging<1.9.0,>=1.8.0 # Apache-2.0
 retrying!=1.3.0,>=1.2.3 # Apache-2.0
-posix-ipc
--- ironic-2015.1.2/requirements.txt.~1~	2015-10-13 10:53:40.000000000 -0700
+++ ironic-2015.1.2/requirements.txt	2016-02-01 20:03:53.251275842 -0800
@@ -5,10 +5,8 @@ pbr!=0.7,<1.0,>=0.6
 SQLAlchemy<=0.9.99,>=0.9.7
 alembic<0.8.1,>=0.7.2
 eventlet!=0.17.0,>=0.16.1
-lxml>=2.3
 WebOb>=1.2.3
 greenlet>=0.3.2
-sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 netaddr>=0.7.12
 paramiko>=1.13.0
 iso8601>=0.1.9
@@ -18,12 +16,10 @@ python-keystoneclient<1.4.0,>=1.2.0
 python-swiftclient<2.5.0,>=2.2.0
 stevedore<1.4.0,>=1.3.0 # Apache-2.0
 pysendfile==2.0.0
-websockify<0.7,>=0.6.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.context<0.3.0,>=0.2.0 # Apache-2.0
 oslo.db<1.8.0,>=1.7.0 # Apache-2.0
-oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
 oslo.i18n<1.6.0,>=1.5.0 # Apache-2.0
 oslo.policy<0.4.0,>=0.3.1 # Apache-2.0
 oslo.serialization<1.5.0,>=1.4.0 # Apache-2.0
@@ -37,4 +33,3 @@ Jinja2>=2.6 # BSD License (3 clause)
 keystonemiddleware<1.6.0,>=1.5.0
 oslo.messaging<1.9.0,>=1.8.0 # Apache-2.0
 retrying!=1.3.0,>=1.2.3 # Apache-2.0
-posix-ipc
